cmd/prowjob-controller/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	controller "github.com/openshift/release-controller/pkg/controller/prowjob"
```

Comment:
You should look into `openshift-goimports` to format these

Reply:
Didn't know tool existed, Thanks @stevekuznetsov!